code/game/objects/items.dm

```diff
@@ -376,6 +376,8 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 			if(51 to INFINITY)
 				. += "[src] is as well weighted as possible for blocking"
 	if(force)
+		if(!force_string)
+			set_force_string()
```

Comment:
I feel like it's really weird place to generate it, shouldn't this be done on init?

Reply:
My reasoning to put it here, rather than on init, is to reduce the total amount of times this would be called.

If first generation is done only when it's examined, then you save having to run this proc for every single item in the game with any force at all.
99% of them are unlikely to be examined in general, wasting some processing time adding strings to them.